test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should be ignored and deleted remotely, Kindly delete it from GitHub as it should be hidden on Github